packages/logging-bunyan/README.md

```diff
@@ -35,19 +29,12 @@ logger.info('shields at 99%');
 
 It's incredibly easy to get authenticated and start using Google's APIs. You can set your credentials on a global basis as well as on a per-API basis. See each individual API section below to see how you can auth on a per-API-basis. This is useful if you want to use different accounts for different Google Cloud services.
 
-### On Google Cloud Platform
+### On Google Compute Engine
```

Comment:
Any reason for the edit? 'Compute Engine' exclude GAE, GKE and Cloud Functions. I think the instructions here are valid for all these environments.